packages/@aws-cdk/cx-api/lib/context/security-group.ts

```diff
@@ -0,0 +1,16 @@
+
+/**
+ * Properties of a discovered SecurityGroup.
+ */
+export interface SecurityGroupContextResponse {
+  /**
+   * The security group's id.
+   */
+  readonly securityGroupId: string;
+
+  /**
+   * Whether the security group of the load balancer allows all outbound
```

Comment:
`SecurityGroup.fromLookup()` can also be called without a LoadBalancer context... what does this property mean then?

Reply:
It's more general-purpose than this comment conveys. I'll adjust the wording so that this is clear.